Update public-nodes.md (#308)
rsksmart/rsksmart.github.io: _rsk/public-nodes.md

---
layout: rsk
title: Using RSK Public Nodes (Mainnet & Testnet) provided by IOVLabs
tags: rsk, networks, versions, rpc, mainnet, testnet, cUrl
description: "RSK Nodes: Public nodes (Mainnet, Testnet), Versioning, RPC Methods, and cUrl example"
collection_order: 2200
---

## Public Nodes

> IOVLabs currently provides two public nodes that you can use for testing purposes and you can find the information bellow. In case you want to run your own RSK Node (highly recommended for production environments in accordance to the bitcoiner's maxim: Don't trust. Verify.) you can follow the <a href="/rsk/node/install/">installation instructions here</a>.

Reviewer: ```suggestion
IOVLabs currently provides two public nodes that you can use for testing purposes and you can find the information below. In case you want to run your own RSK Node (highly recommended for production environments in accordance with the bitcoiners' maxim: Don't trust. Verify.), you can follow the [installation instructions here](/rsk/node/install/).
```

Suggestion (1) : Minimal edit.

Reviewer: ```suggestion
IOVLabs currently provides two public nodes that you can use
for testing purposes, and you will find that information below.

Alternatively, follow the [installation instructions](/rsk/node/install/),
to run your own RSK node.
This is highly recommended for production environments,
and in accordance with the bitcoiners' maxim: **Don't trust. Verify.**

```

Suggestion (2): Edit with more changes (recommended)

I've rearranged the sentence to give the bitcoiners' maxim **more emphasis**, plus take it out of the parentheses to avoid confusing punctuation.
